src/content/en/2020/resource-hints.md

```diff
@@ -199,9 +205,10 @@ Be mindful that fonts preloaded without the `crossorigin` attribute will be fetc
 
 When it's time to choose a resource for use with different screen sizes, reach for the `media` attribute with `preload` to optimize your media queries.
 
+{# TODO(editors): Should we keep this shorter to prevent small amounts of scrolling? #}
```

Comment:
@rviscomi

Reply:
This might also be a question for @Zizzamia because desktop/mobile.css may be relevant to the example.

Generally, I'm ok with code scrolling, we have an overflow style to handle it.

Reply:
The reason I suggest removing the scroll from here is because it is so small it's almost distracting and has little to no useful content behind the scroll:
![image](https://user-images.githubusercontent.com/6392995/101592650-7f1d2100-39a3-11eb-80a2-cfe106f7d2a4.png)

Only these 2 characters are hidden.

I just find it more aesthetic to not have scrolling for something this minor.

Reply:
The scroll area would depend on users' viewport size so it may be more or less. I wouldn't worry about it.